src/app/globalStatusBar/userSetting/userSettingDialog.js

```diff
@@ -52,17 +75,25 @@ class UserSettingDialog extends Component {
                     }
                 >
                     <DialogTitle>
-                        User settings{' '}
-                        {this.props.user ? `for ${this.props.user}` : ''}
+                        {this.props.user
+                            ? (
+                                <>
+                                    <LocalizedMessage id="globalStatusBar.userSetting.dialog.title.user-present"/>
+                                    {this.props.user}
+                                </>
+                            )
+                            : (
+                                <LocalizedMessage id="globalStatusBar.userSetting.dialog.title.user-absent"/>
+                            )}
                     </DialogTitle>
                     <DialogContent>
                         <AppBar position="static">
                             <Tabs
                                 value={this.state.currentTab}
                                 fullWidth
-                                onChange={this.handleChange}
-                            >
-                                <Tab label="Password" />
+                                onChange={(e, v) => this.setState({ currentTab: v })}>
```

Comment:
Descriptive please.

Reply:
Addressed in 03fa3e55e8bfcede99740d7bc85cc3fb0222eece.